tools/cdata.js

```diff
@@ -116,6 +116,14 @@ function writeHtmlGzipped(sourceFile, resultFile, page) {
     html = filter(html, "html-minify-ui");
     console.info("Minified to " + html.length + " characters");
 
+    //Set to true to output minified HTML to filename_minified.htm for testing and validation
```

Comment:
Even though the output is optional I will not approve this change to be included in the final (published) version.
You may have your copy of cdata.json as you like, but master will remain unchanged. The only addition will be inclusion of pixart file.